[2024-04-29T05:37:50.581+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: etl.get_files scheduled__2024-04-07T00:00:00+00:00 [queued]>
[2024-04-29T05:37:50.705+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: etl.get_files scheduled__2024-04-07T00:00:00+00:00 [queued]>
[2024-04-29T05:37:50.721+0000] {taskinstance.py:2193} INFO - Starting attempt 1 of 1
[2024-04-29T05:37:52.088+0000] {taskinstance.py:2217} INFO - Executing <Task(PythonOperator): get_files> on 2024-04-07 00:00:00+00:00
[2024-04-29T05:37:52.172+0000] {standard_task_runner.py:60} INFO - Started process 1972 to run task
[2024-04-29T05:37:52.256+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'etl', 'get_files', 'scheduled__2024-04-07T00:00:00+00:00', '--job-id', '12', '--raw', '--subdir', 'DAGS_FOLDER/etl.py', '--cfg-path', '/tmp/tmph42b0nlm']
[2024-04-29T05:37:52.295+0000] {standard_task_runner.py:88} INFO - Job 12: Subtask get_files
[2024-04-29T05:37:52.862+0000] {task_command.py:423} INFO - Running <TaskInstance: etl.get_files scheduled__2024-04-07T00:00:00+00:00 [running]> on host 422c09390d6c
[2024-04-29T05:37:53.623+0000] {taskinstance.py:2513} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='etl' AIRFLOW_CTX_TASK_ID='get_files' AIRFLOW_CTX_EXECUTION_DATE='2024-04-07T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-04-07T00:00:00+00:00'
[2024-04-29T05:37:53.704+0000] {etl.py:24} INFO - 5 files found in /opt/***/dags/data
[2024-04-29T05:37:53.734+0000] {python.py:202} INFO - Done. Returned value was: ['/opt/***/dags/data/github_events_02.json', '/opt/***/dags/data/github_events_03.json', '/opt/***/dags/data/github_events_04.json', '/opt/***/dags/data/github_events_05.json', '/opt/***/dags/data/github_events_01.json']
[2024-04-29T05:37:53.984+0000] {taskinstance.py:1149} INFO - Marking task as SUCCESS. dag_id=etl, task_id=get_files, execution_date=20240407T000000, start_date=20240429T053750, end_date=20240429T053753
[2024-04-29T05:37:54.157+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-04-29T05:37:54.269+0000] {taskinstance.py:3312} INFO - 0 downstream tasks scheduled from follow-on schedule check
